code/datums/mind.dm

```diff
@@ -1031,12 +1031,12 @@
 			if("clear")
 				remove_antag_datum(/datum/antagonist/changeling)
 				special_role = null
-				to_chat(current, "<span class='userdanger'>You grow weak and lose your powers! You are no longer a changeling and are stuck in your current form!</span>")
+				to_chat(current, "<span class='userdanger'>We grow weak and lose our powers! You are no longer a changeling and are stuck in your current form!</span>")

```

Comment:
Wouldn't it be "We are no longer a changeling"?

Reply:
he's not a changeling

Reply:
no, this entire line should be reverted, it's happened AFTER you're no longer a changeling, thus it SHOULD be You and Your

Reply:
Ok